src/Life/Main/Pull.hs

```diff
@@ -0,0 +1,34 @@
+-- | Command to update local state from remote state.
+
+module Life.Main.Pull
+       ( lifePull
+       ) where
+
+import Path (Dir, File, Path, Rel)
+
+import Life.Github (Owner, cloneRepo, updateFromRepo)
+import Life.Main.Init (lifeInitQuestion)
+import Life.Message (abortCmd, choose, warningMessage)
+import Life.Shell (LifeExistence (..), whatIsLife)
+
+lifePull :: Owner -> Set (Path Rel File) -> Set (Path Rel Dir) -> IO ()
+lifePull owner withoutFiles withoutDirs = whatIsLife >>= \case
+    OnlyRepo _ -> warningMessage ".life file not found" >> update
```

Comment:
Should `update` be the default action when `.life` is not found ( I mean `life` file to be copied from the repo)?
And after update the action should take place?